doc/agent.rst

```diff
@@ -10,7 +10,9 @@ Initialization process
 The agent tries to be as autonomous as possible. One of the goal is to ship
 agents without requiring external provisioning tools, such as Chef or Puppet.
 Therefore, the agent attempts to install itself as a service, and also supports
-a builtin upgrade protocol (described in the next section).
```

Comment:
Yeah; I believe PR catches all the references to the upgrade module in docs, but we probably still have some code around it.